include/core-image-renesas-base.inc

```diff
@@ -35,7 +35,148 @@ IMAGE_BOOT_FILES:rzg2l-sbc = " \
     target/images/dtbs/overlays/*;dtb/renesas/overlays/ \
     target/images/Image;Image \
     target/env/uEnv.txt;uEnv.txt \
-    target/env/Readme.md;Readme.md \
+    Readme.md;Readme.md \
 "
 
 DEPENDS += " linux-yocto uenv"
+
+IMAGE_LINK_NAME = "${IMAGE_BASENAME}${IMAGE_MACHINE_SUFFIX}"
+
+add_bootloader_rootfs() {
+    install -d ${IMAGE_ROOTFS}/boot/uload-bootloader
+    cp -rf ${DEPLOY_DIR_IMAGE}/target/images/bl2_bp-rzg2l-sbc.bin ${IMAGE_ROOTFS}/boot/uload-bootloader
+    cp -rf ${DEPLOY_DIR_IMAGE}/target/images/fip-rzg2l-sbc.bin ${IMAGE_ROOTFS}/boot/uload-bootloader
+}
+
+# Clean the output directory after the build
+do_release_clean() {       
+    rm -f ${DEPLOY_DIR_IMAGE}/Image*
+    rm -f ${DEPLOY_DIR_IMAGE}/rzpi*
+    rm -f ${DEPLOY_DIR_IMAGE}/${IMAGE_BASENAME}*
+    rm -f ${DEPLOY_DIR_IMAGE}/Readme.md
```

Comment:
It seems you removed the command to clean U-Boot, ATF, and Flashwriter files. Is that correct? Did you perform a clean build and verify that the build folder is completely clean?